src/_locales/en_US/messages.json

```diff
@@ -512,5 +512,37 @@
                 "example": "Facebook Like"
             }
         }
+    },
+    "sync_intro": {
+        "message": "Cloud syncing:<ul><li>Requires Firefox/Chrome Sync</li><li>Download combines the lists of sites where your Badger is disabled</li><li>Upload overwrites the list saved in the cloud</li></ul>",
+        "description": "A brief explanation of how syncing works. Shown above the upload/download disabled sites buttons under the Manage Data options page tab."
+    },
+    "upload": {
+        "message": "Upload",
+        "description": "Button label for Sync Upload on Manage Data options page tab."
+    },
+    "upload_whitelist": {
```

Comment:
We should rename all of these message keys to say "cloud" instead of "whitelist" since we may choose to upload more than just the list of disabled sites in the future.

Reply:
done